core/src/main/java/com/rudderstack/android/sdk/core/RudderPreferenceManager.java

```diff
@@ -10,7 +10,8 @@ class RudderPreferenceManager {
     private static final String RUDDER_SERVER_CONFIG_KEY = "rl_server_config";
     private static final String RUDDER_SERVER_CONFIG_LAST_UPDATE_KEY = "rl_server_last_updated";
     private static final String RUDDER_TRAITS_KEY = "rl_traits";
-    private static final String RUDDER_APPLICATION_INFO_KEY = "rl_application_info_key";
+    private static final String RUDDER_APPLICATION_BUILD_KEY = "rl_application_build_key";
```

Comment:
Don't change the key. It will lead to bug.

Reply:
Added the support for migrating the versionCode to build number